index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <meta charset="utf-8" />
+      <title>Carrera Programmer - Portfolio</title>
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,100;1,300;1,700&family=Roboto:wght@100;300&display=swap" rel="stylesheet">
+      <link rel="stylesheet" href="styles.css">
+      <script src="https://kit.fontawesome.com/321e54ee43.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+      <header>
+        <nav class="navbar">
+            <p class="logo">Carrera<i  id=logo-dat class="fa-solid fa-circle-dot"></i>Programmer</p>
+            <i id="burguer-menu" class="fa-solid fa-bars"></i>
+        </nav>
+        <section class="welcome-section">
+            <h1>Hey There<span>.</span><br> I'm Carrera</h1>
+            <h2>I'm a Full Stack Developer</h2>
+            <p>I can print your dreams in a web page. If you like what you see and have a project you need coded, don't hestiate to contact me</p>
+            <div class="social-icons">
+                <i class="fa-brands fa-github-alt"></i>
+                <i class="fa-brands fa-linkedin-in"></i>
+                <i class="fa-brands fa-twitter"></i>
+            </div>
```

Comment:
- [ ] The list of icons should use a `<ul>` tag to group the icons, and `<li>` per icon, then each icon should be a link by using `<a>` tag
e.g: `<li><a href="url">icon</a></li>`